File: cs/Iris/ParserUtils/LexParseException/NewLineInStringException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Iris.ParserUtils.LexParseException
{
    public class NewLineInstringException : BaseException
    {
        public NewLineInstringException(int lNum, int cNum)
            : base(lNum, cNum)
        {

        }

        public override string ExceptionToString()
        {
            StringBuilder str = new StringBuilder();

            str.Append("unexpected new line in string: ");
            str.Append(base.stringTrace());

            return str.ToString();
        }
    }
}

File: java/Iris/src/com/rokuan/iris/exception/NewLineInStringException.java
package com.rokuan.iris.exception;

public class NewLineInStringException extends BaseException
{
	public NewLineInStringException(int lNum, int cNum)
	{
		super(lNum, cNum);
	}

	public String ExceptionToString()
	{
		StringBuilder str = new StringBuilder();

		str.append("unexpected new line in string: ");
		str.append(super.stringTrace());

		return str.toString();
	}
}